dnf5-plugins/copr_plugin/json.hpp

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef DNF5_COMMANDS_COPR_JSON_HPP
+#define DNF5_COMMANDS_COPR_JSON_HPP
+
+#include "libdnf/base/base.hpp"
+
+#include <json.h>
+
+class Json {
+private:
+    bool cleanup;
+    struct json_object * root;
```

Comment:
I asked on IRC without any answer (praiskup> Hi, is there some tooling in dnf5 (usable from a plugin) that would allow me to download a json file into memory and parse?)

This json-c wrapper seemed pretty trivial to implement, so I did it (without thinking too much about the API).  Lemme know if it deserves some tweaks to make it more useful.

Reply:
I would prefer to use some external C/C++ library over having to maintain our own implementation.
But I haven't done any thorough investigation so I do not know what is available. Maybe `rapidjson-devel` package which seems to be part of RHEL buildroot?

Reply:
I'd probably prefer keeping this trivial 50-liner (when already done), because additional dep might easily become a valid reason to not move the "copr" plugin into the set of built-in commands in the future.  Just my preference.